azure_functions_worker/bindings/meta.py

```diff
@@ -300,6 +300,12 @@ def deferred_bindings_decode(binding: typing.Any,
     """
     global deferred_bindings_cache
 
+    # Only applies to Event Hub and Service Bus - cannot cache
```

Comment:
Lets change this to
if Source is  "AzureServiceBusReceivedMessage" or "AzureEventHubsEventData", skip caching and add a comment why